src/nginxconfig/i18n/fr/templates/domain_sections/https.js

```diff
@@ -32,6 +32,8 @@ export default {
     enableHttp2Connections: `${common.enable} les connexions ${common.http}/2`,
     http3: `${common.http}/3`,
     enableHttp3Connections: `${common.enable} les connexions ${common.http}/3`,
+    portReuse: 'My English string', // TODO: translate
```

Comment:
He means have this as the English translation haha. So "Reuseport" not actually "My English string"
Numpty

Reply:
oooooooooooooow

Reply:
:lul:

Reply:
mean :panspoonsad: